wurst/closures/ClosureKeyPresses.wurst

```diff
@@ -0,0 +1,106 @@
+package ClosureKeyPresses
+import HashMap
+
+/*  
+    Example usage:
+        onKeyPress(OSKEY_A) -> 
+            print("A player pressed on the A key.")
+
+        onKeyPress(OSKEY_B, OSKEY_META.SHIFT) -> 
+            print("A player pressed on the B key while holding a SHIFT key.")
+
+        onKeyPress(OSKEY_C, false) -> 
```

Comment:
Seems a little weird, why not `onKeyRelease` ?

Reply:
good idea